src/components/HomePage.vue

```diff
@@ -136,6 +142,22 @@
     toggleToDoEditState();
   }
 
+  function filterToDos(filterCategory: string, order: number) {
+    if (filterCategory === '') {
+      return;
+    }
+    const currentToDosOnDisplay = displayItems.value.slice();
+    if (filterCategory === 'Title') {
```

Comment:
Create an enum for the filtering properties, you are using them in multiple places